### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
first commit

### CODE REVIEW

File: src/arrayMethodJoin.js [hunk 1]
```diff
@@ -3,9 +3,28 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
```
Comment (Bohdan-Maksymiuk) [inline src/arrayMethodJoin.js:9]: remove this string

File: src/arrayMethodJoin.js [hunk 2]
```diff
@@ -3,9 +3,28 @@
...
+    let string = '';
+    let insert = '';
```
Comment (Bohdan-Maksymiuk) [inline src/arrayMethodJoin.js:11]: you may declare it into the loop and this empty variable will be created on every iteration

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
all suggestions are implemented

**Comment by pavel-gutsal** [comment]
commented text removed

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**